src/app/court/court.component.ts

```diff
@@ -1,14 +1,185 @@
 import { Component, OnInit } from '@angular/core';
+import {FormControl} from '@angular/forms';
+
+import {Observable} from 'rxjs/Observable';
+import 'rxjs/add/operator/startWith';
+import 'rxjs/add/operator/map';
+import {MatAutocompleteSelectedEvent, MatDialog, MatRadioChange} from '@angular/material';
+
+import * as _ from 'lodash';
```

Comment:
Instead of import all of lodash, you can just import the function(s) you use. e.g. 
`import {find} from 'lodash'`